src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -767,6 +775,7 @@ private void commitFile(String encodedPartition) {
     startOffsets.remove(encodedPartition);
     offsets.remove(encodedPartition);
     offset = offset + recordCounter;
+    committedOffset = offset;
```

Comment:
same here? `committedOffset` is `offset - 1`.

Reply:
I was thinking the same thing - `endOffset` seems to be more accurate.

Reply:
I think the choice here is, do we want to report the most accurate number or the most safest number. (`offset-1` vs `endOffset`. I think that's what it is here, but I might have to look closer). Maybe go with the safest option of `endOffset`?